kinetic/distribution.yaml

```diff
@@ -5980,6 +5980,25 @@ repositories:
       url: https://github.com/ros-drivers/linux_peripheral_interfaces.git
       version: kinetic
     status: maintained
+  livox_ros_driver:
+    doc:
+      type: git
+      url: https://github.com/Livox-SDK/Livox-SDK-ROS.git
+      version: master
+    release:
+      packages:
+      - display_hub_points
+      - display_lidar_points
+      - linux_ros_driver
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/Livox-SDK/Livox-SDK-ROS.git
```

Comment:
This is not a release repository generated by bloom. This whole stanza should be generated by bloom when making a release: http://wiki.ros.org/bloom/Tutorials/FirstTimeRelease please remove it for now if you just want to index your source code.

Reply:
Thanks, tfoote!